.github/workflows/test.yml

```diff
@@ -141,10 +129,10 @@ jobs:
         run: make local-kc-run
 
       - name: Add Sites
-        run: cd server && POSTGRES_HOST=localhost npm run feed-sites sites.xlsx
+        run: cd server && POSTGRES_HOST=localhost npm run feed-sites
```

Comment:
I think I'm confused about what we need now. Feed data should now be all encompassing right? Don't we want to just consistently feed data on both cypress and jest?

Reply:
I would say that's ideal, yes - I was aiming not to disrupt the test behaviour too much, but you know what, let's just remove all automated use of feed-sites and feed-participants. Much simpler and more predictable that way.